<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationWidget</class>
 <widget class="QWidget" name="SegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>233</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>251</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tissue Segmentation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>233</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Thresholding</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QSlider" name="lowThresholdSlider">
            <property name="toolTip">
             <string>Adjusts lower threshold.</string>
            </property>
            <property name="statusTip">
             <string>Adjusts lower threshold.</string>
            </property>
            <property name="minimum">
             <number>-1500</number>
            </property>
            <property name="maximum">
             <number>7000</number>
            </property>
            <property name="pageStep">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="highThresholdSlider">
            <property name="toolTip">
             <string>Adjusts higher threshold.</string>
            </property>
            <property name="statusTip">
             <string>Adjusts higher threshold.</string>
            </property>
            <property name="minimum">
             <number>-1500</number>
            </property>
            <property name="maximum">
             <number>7000</number>
            </property>
            <property name="pageStep">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="highSpinBox">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Adjusts higher threshold.</string>
            </property>
            <property name="statusTip">
             <string>Adjusts higher threshold.</string>
            </property>
            <property name="minimum">
             <number>-1500</number>
            </property>
            <property name="maximum">
             <number>7000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="lowSpinBox">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Adjusts lower threshold.</string>
            </property>
            <property name="statusTip">
             <string>Adjusts lower threshold.</string>
            </property>
            <property name="minimum">
             <number>-1500</number>
            </property>
            <property name="maximum">
             <number>7000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <widget class="QCheckBox" name="checkBoxApplyThresholds">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Applies coloring of data according to thresholds.</string>
              </property>
              <property name="statusTip">
               <string>Applies coloring of data according to thresholds.</string>
              </property>
              <property name="text">
               <string>Apply Thresholds</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonColor">
              <property name="toolTip">
               <string>Sets color of thresholded data.</string>
              </property>
              <property name="statusTip">
               <string>Sets color of thresholded data.</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="buttonPickLo">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Picks lower threshold from data by clicking with left mose button.</string>
            </property>
            <property name="statusTip">
             <string>Picks lower threshold from data by clicking with left mose button.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/resources/dropper.png</normaloff>:/icons/resources/dropper.png</iconset>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="buttonPickHi">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Picks higher threshold from data by clicking with left mose button.</string>
            </property>
            <property name="statusTip">
             <string>Picks higher threshold from data by clicking with left mose button.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/resources/dropper.png</normaloff>:/icons/resources/dropper.png</iconset>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QPushButton" name="pushButtonSetToActiveRegion">
            <property name="text">
             <string>Set to Active Region</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Model Creation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonCreateSurfaceModel">
            <property name="text">
             <string>Create Surface Model</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>12</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
